atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/metrics/SweepMetricsManager.java

```diff
@@ -15,42 +15,43 @@
  */
 package com.palantir.atlasdb.sweep.metrics;
 
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.MetricRegistry;
 import com.palantir.atlasdb.AtlasDbMetricNames;
 import com.palantir.atlasdb.keyvalue.api.SweepResults;
-import com.palantir.atlasdb.keyvalue.api.TableReference;
-import com.palantir.atlasdb.logging.LoggingArgs;
+import com.palantir.atlasdb.util.MetricsManager;
 
 // Not final for Mockito
 @SuppressWarnings("checkstyle:FinalClass")
 public class SweepMetricsManager {
-    private final SweepMetricsFactory factory = new SweepMetricsFactory();
-
-    private final SweepMetric<Long> cellsExamined = factory.accumulatingLong(AtlasDbMetricNames.CELLS_EXAMINED);
-    private final SweepMetric<Long> cellsDeleted = factory.accumulatingLong(AtlasDbMetricNames.CELLS_SWEPT);
-    private final SweepMetric<Long> timeSweeping = factory.simpleLong(AtlasDbMetricNames.TIME_SPENT_SWEEPING);
-    private final SweepMetric<Long> totalTime = factory.simpleLong(AtlasDbMetricNames.TIME_ELAPSED_SWEEPING);
-    private final SweepMetric<String> tableSweeping = factory.simpleString(AtlasDbMetricNames.TABLE_BEING_SWEPT);
-    private final SweepMetric<Long> sweepErrors = factory.simpleMeter(AtlasDbMetricNames.SWEEP_ERROR);
-
-    public void resetBeforeDeleteBatch() {
-        cellsExamined.set(0L);
-        cellsDeleted.set(0L);
-    }
+    public static final String METRIC_BASE_NAME = SweepMetricsManager.class.getPackage().getName() + ".SweepMetric";
+
```

Comment:
This is an effort to keep same metric names without the actual `SweepMetric` class. It may make debugging a bit harder though, I can either create a SweepMetric class and store meters there, or change metric names again.